apis/core/lib/auth.ts

```diff
@@ -56,7 +63,7 @@ export default async () => {
     const response = await fetch(`${env.AUTH_ISSUER}/.well-known/openid-configuration`)
     if (response.ok) {
       const oidcConfig = await response.json()
-      router.use(createJwtHandler(true, oidcConfig.jwks_uri))
+      router.use(createJwtHandler(env.production, oidcConfig.jwks_uri))
```

Comment:
Had to change here because a hard `true` rejects the request immediately when there is no token. Disabling this in the dev environment lets the auth backdoor to kick in.